csharp/ql/src/Telemetry/ExternalApi.qll

```diff
@@ -104,8 +105,14 @@ class ExternalApi extends DotNet::Callable {
   pragma[nomagic]
   predicate isSink() { sinkNode(this.getAnInput(), _) }
 
+  /** Holds if this API is a known neutral. */
+  pragma[nomagic]
+  predicate isNeutral() { this instanceof FlowSummaryImpl::Public::NeutralCallable }
+
   /** Holds if this API is supported by existing CodeQL libraries, that is, it is either a recognized source or sink or has a flow summary. */
```

Comment:
Should this comment be updated to reflect the additional condition?

Reply:
Yes, definitely! Thank you!